<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1360</width>
      <height>660</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonx_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect about x = 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttony_0">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect about y = 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonx_y">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>670</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect about x = y</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>670</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>670</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelY">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>670</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>670</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate about      x:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>670</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>670</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csabButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>670</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear screen and buffer</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>670</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Î¸:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTheta">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>670</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
